### PR TITLE
Added Api routes 

### PR DESCRIPTION
**Pull request by twentymurial33**
Added API Routes and now working on adding routes to render the data stored in prisma to the FE

### CODE REVIEW

File: client/src/pages/Details.js [hunk 1]
```diff
@@ -76,5 +76,14 @@ const Container = styled.div`
...
 `;
+//added this here but not really sure what its doing--- found in docs
+export async function getServerSideProps() {
+  // Fetch data from external API
+  const res = await fetch(`/api/food`);
+  const feed = await res.json();
+
+  // Pass data to the page via props
+  return { props: { feed } };
+}
```
Comment (brentmclark) [inline client/src/pages/Details.js:87]: `getServerSideProps` is a `Next.js` function that facilities the assembly of props on the server prior to creating the HTML markup for the initial page.  You can read more about it [here](https://nextjs.org/docs/basic-features/data-fetching/get-server-side-props)

Ultimately, you're not using `Next.js` in this project so you don't need it here.

```suggestion
```

File: client/src/pages/api/food/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  const { img, author, title } = req.body;
+  const foodItem = await prisma.food.create({
+    data: {
+      title: title,
+      author: author,
+      img: img,
+    },
+  });
+  res.json(foodItem);
+}
```
Comment (brentmclark) [inline client/src/pages/api/food/index.js:13]: I think it's time to focus on the app's folder structure for a moment.  This code resides under a root-level folder named `client`, but the code is for the server.  

How do you feel about adding a new folder structure like this:


-client
-server
--api
---food.js

`food.js` would then contain methods that map to the HTTP verbs you're using.  for example `post`, `put`, etc.

File: client/src/pages/api/update/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  const foodItem = await prisma.food.update({
+    where: { id: Number(id) },
+    data: {
+      img,
+      author,
+      title,
+    },
+  });
+  res.json(foodItem);
+}
```
Comment (brentmclark) [inline client/src/pages/api/update/index.js:15]: See comment above regarding folder structure and naming conventions.

Comment (twentymurial33) [reply client/src/pages/api/update/index.js:15]: yes

**Comment by twentymurial33** [comment]
I changed the folder structure - thank you - now running into a new error with http://localhost:5000

**Review by brentmclark — APPROVED** [review]
This code works, but I think we should clean up the API folder and file structure a bit more with this PR or one in the near future.  You have two files across two folders that operate on the `food` model.  One is organized under the `food` folder while the other is in a folder named `update`.  This disagreement in naming convention suggests inconsistency in the approach that we can and should address before it gets out of hand.  

Maybe a

**Pull request merged**